roles/nextflow/templates/nextflow_miarka_site.config.j2

```diff
@@ -1,6 +1,10 @@
 
 cleanup = true
 
+{% if site == "upps" %}
+    cleanup = false
```

Comment:
```suggestion
cleanup = false
{% else %}
cleanup = true
{% endif %}
```

Even though I think the last occurrence of a parameter is used, I think we should avoid having two occurrences of the `cleanup` parameter. I would suggest to remove line 2 in this template and instead set `cleanup = True` in an else statement. Something like the above.

Reply:
Good suggestion! I have updated the if statement accordingly.